src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -3005,7 +3007,7 @@ module Util =
                 | Fable.DeclaredType(entRef, _) ->
                     let ent = com.GetEntity(entRef)
                     if ent.IsInterface then
-                        let entNs, entName = splitNameSpace ent.FullName
+                        let entName = getEntityFullName com ctx entRef
```

Comment:
@alexswan10k Do we need to split the full name here? Not saying we do, just asking.

Reply:
I am not sure. Doesn't this just give you the imported alias name (or type name if no alias present)? We do have some test coverage for this area and everything passes. There could well be holes though.